<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshViewerTabWidget</class>
 <widget class="QWidget" name="MeshViewerTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <widget class="QFrame" name="frameToolbox">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frameVTK">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnZoomIn">
       <property name="toolTip">
        <string>Zoom in</string>
       </property>
       <property name="text">
        <string>Zoom in</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/zoom-in.svg</normaloff>:/meshviewer/images/zoom-in.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl++</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnZoomOut">
       <property name="toolTip">
        <string>Zoom out</string>
       </property>
       <property name="text">
        <string>Zoom out</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/zoom-out.svg</normaloff>:/meshviewer/images/zoom-out.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+-</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnResetCamera">
       <property name="toolTip">
        <string>Reset camera</string>
       </property>
       <property name="text">
        <string>Reset camera</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/zoom-fit-best.svg</normaloff>:/meshviewer/images/zoom-fit-best.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnX">
       <property name="toolTip">
        <string>Set view direction to +X</string>
       </property>
       <property name="text">
        <string>Set view direction to +X</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/x+.png</normaloff>:/meshviewer/images/x+.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnX_">
       <property name="toolTip">
        <string>Set view direction to -X</string>
       </property>
       <property name="text">
        <string>Set view direction to -X</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/x-.png</normaloff>:/meshviewer/images/x-.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnY">
       <property name="toolTip">
        <string>Set view direction to +Y</string>
       </property>
       <property name="text">
        <string>Set view direction to +Y</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/y+.png</normaloff>:/meshviewer/images/y+.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnY_">
       <property name="toolTip">
        <string>Set view direction to -Y</string>
       </property>
       <property name="text">
        <string>Set view direction to -Y</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/y-.png</normaloff>:/meshviewer/images/y-.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnZ">
       <property name="toolTip">
        <string>Set view direction to +Z</string>
       </property>
       <property name="text">
        <string>Set view direction to +Z</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/z+.png</normaloff>:/meshviewer/images/z+.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnZ_">
       <property name="toolTip">
        <string>Set view direction to -Z</string>
       </property>
       <property name="text">
        <string>Set view direction to -Z</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/z-.png</normaloff>:/meshviewer/images/z-.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnShowAxes">
       <property name="toolTip">
        <string>Show orientation axes</string>
       </property>
       <property name="text">
        <string>Show orientation axes</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/xyz2.png</normaloff>:/meshviewer/images/xyz2.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnScreenShot">
       <property name="toolTip">
        <string>Take a screenshot</string>
       </property>
       <property name="text">
        <string>Take a screenshot</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/camera.svg</normaloff>:/meshviewer/images/camera.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Shift+C</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnLight">
       <property name="toolTip">
        <string>Show light</string>
       </property>
       <property name="text">
        <string>Show light</string>
       </property>
       <property name="icon">
        <iconset resource="meshviewer_res.qrc">
         <normaloff>:/meshviewer/images/light.svg</normaloff>:/meshviewer/images/light.svg</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+L</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="meshviewer_res.qrc"/>
 </resources>
 <connections/>
</ui>
